sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -192,6 +192,8 @@ private void startTracing(final @NotNull Activity activity) {
 
         final TransactionOptions transactionOptions = new TransactionOptions();
         if (options.isEnableActivityLifecycleTracingAutoFinish()) {
+          transactionOptions.setDeadlineTimeout(
+              TransactionOptions.SENTRY_AUTO_TRANSACTION_DEADLINE_MS);
```

Comment:
deadline should be set even if `isEnableActivityLifecycleTracingAutoFinish` is false, to be consistent with the other transactions